Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -62,6 +66,50 @@ theorem tendstoInMeasure_iff_norm [SeminormedAddCommGroup E] {l : Filter ι} {f
   simp_rw [TendstoInMeasure, dist_eq_norm]
 #align measure_theory.tendsto_in_measure_iff_norm MeasureTheory.tendstoInMeasure_iff_norm
 
+-- The ae-limit is ae-unique.
+theorem ae_unique_of_ae_limit {α ι E : Type*} [TopologicalSpace E] [T2Space E]
+    {x : MeasurableSpace α} {μ : Measure α} {g h : α → E} {f : ι → α → E} {l : Filter ι} [l.NeBot]
+    (hg : ∀ᵐ ω ∂μ, Filter.Tendsto (fun i => f i ω) l (nhds (g ω))) (hh :
```

Comment:
Could you use `𝓝` instead of `nhds` (obtained by typing \nhds) here and elsewhere? Also, your assumption `hh` is split among two lines, while it would fit on one line.